docs/userguide.rst

```diff
@@ -199,12 +199,14 @@ workflow workspace via the ``open`` command.
 .. code-block:: console
 
    $ reana-client open -w myanalysis.1 jupyter
-   http://reana.cern.ch:30080/7cd4d23e-48d1-4f7f-8a3c-3a6d256fb8bc?token=P-IkL_7w25IDHhes8I7DtICWLNQm2WAZ9gkoKC2vq10
+   https://reana.cern.ch/7cd4d23e-48d1-4f7f-8a3c-3a6d256fb8bc?token=P-IkL_7w25IDHhes8I7DtICWLNQm2WAZ9gkoKC2vq10
    It could take several minutes to start the interactive session.
 
 Open the link returned by the command in order to access the interactive
```

Comment:
Expand last phrase to stress the strengths:

REANA currently supports Jupyter notebooks.  Note that you can pass any notebook image you are interested to run on the workspace, such as PySpark, or even your own image, by using the ``--image`` option.